bip-update-process.md

```diff
@@ -0,0 +1,646 @@
+evidence of it being in active use, and its authors do not object or fail to respond, unless a new author volunteers
+within four weeks.
+
+##### Active ↦ Abandoned
+
+A BIP may evolve from Active to Abandoned when it is no longer in active use. Any community member may initiate this
+Status update by announcing it to the mailing list, and proceed if no objections have been raised for four weeks.
+
+##### Abandoned ↦ Draft
+
+The Abandoned status is generally intended to be a final status for BIPs.
+
+If a BIP Author decides to make another attempt at a previously abandoned proposal, it is generally recommended to
+create a new proposal. (Obviously, the author may borrow any amount of inspiration or actual text from any prior BIPs as
+licensing permits.) The author should take special care to address the issues that caused the prior attempt’s
+abandonment. Even if the prior attempt had been assigned a number, the new BIP will generally be assigned a distinct
+number, unless it is obvious that the new attempt directly continues work on the same idea, in which case it may be
+reasonable to return the Abandoned BIP to Draft status.
+
+### Adoption of proposals
+
+The BIPs repository does not track the sentiment on proposals and does not track the adoption of BIPs beyond whether
+they are in active use or not. It is not intended for BIPs to list additional implementations beyond the reference
+implementation: the BIPs repository is not a signpost where to find implementations.[^OtherImplementations] After a BIP
+is advanced to Proposed, it is up to the bitcoin community to evaluate, adopt, ignore, or reject a BIP. Individual
+bitcoin projects are encouraged to publish a list of BIPs they implement. A good example of this at the time of writing
+this BIP can be observed in [Bitcoin Core’s doc/bips.md
+file](https://github.com/bitcoin/bitcoin/blob/master/doc/bips.md).
+
+### Transferring BIP Ownership
+
+It occasionally becomes necessary to transfer ownership of BIPs to a new author. In general, we would like to retain
+the original author as a co-author of the transferred BIP, but that is up to the original author. A good reason to
+transfer ownership is because the original author no longer has the time or interest in updating it or following through
+with the BIP process, or has fallen off the face of the 'net (i.e. is unreachable or not responding to email). A bad
+reason to transfer ownership is because someone doesn't agree with the direction of the BIP. We try to build consensus
+around a BIP, but if that's not possible, rather than fighting over control, the dissenters should supply a competing
+BIP.
+
+If someone is interested in assuming ownership of a BIP, they should send an email asking to take over, addressed to the
+original author and the BIP editors. If the author does not respond to email in a timely manner, the BIP editors will
+make a unilateral decision (which may be amended should the original author make a delayed reappearance).
```

Comment:
Perhaps they should send to the original author and the list? (We use the list for other announcements, so I think it makes sense to use it here too). Perhaps the original  author has simply changed their email address, or someone on the list can point out how to reach the original author.

Want to specify a deadline here too?

Reply:
Oh yeah, absolutely the list should be part of that process. Thanks for catching that!